### PR TITLE
g

### PR DESCRIPTION
**Pull request by rijad90**
Link to my deployed project:

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/components/GetThoughts.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+
+const GetThoughts = ({ thougts, onPostLike }) => {
+  return (
+    <>
+      {thougts.map((data) => (
+        <div className="thoughts" key={data._id}>
+          <p>{data.message}</p>
+          <div className="likes">
+            <button
```
Comment (Dopest0727) [inline code/src/components/GetThoughts.js:12]: Map for the win

File: code/src/components/GetThoughts.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+              className="heartbutton"
+              onClick={() => onPostLike(data._id)}
+            >
+              <span role="img" aria-label="Heart">
+          ❤️
+        </span>
```
Comment (Dopest0727) [inline code/src/components/GetThoughts.js:18]: Great to see that you solve the issue with the span

File: code/src/components/GetThoughts.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+              
+            </button>
+             x{data.hearts}
+            <div className="date">
+              <p>{formatDistanceToNow(new Date(data.createdAt))} ago</p>
+            </div>
```
Comment (Dopest0727) [inline code/src/components/GetThoughts.js:24]: Personally would have gone with 'moment' but happy it worked

File: code/src/components/HappyThoughts.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
+import React, { useState, useEffect } from "react"
+import GetThoughts from "./GetThoughts"
+import PostThoughts from "./PostThoughts"
+// import {formatRelative } from 'date-fns'
+
+const API_THOUGHTS = "https://happy-thoughts-technigo.herokuapp.com/thoughts"
+const API_LIKES = (thoughtId) =>
+  `https://happy-thoughts-technigo.herokuapp.com/thoughts/${thoughtId}/like`
+
```
Comment (Dopest0727) [inline code/src/components/HappyThoughts.js:9]: Nice and clean

File: code/src/components/HappyThoughts.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+const HappyThougts = () => {
+  /* GET THOUGHTS AND LIKES */
+  const [thougts, setThougts] = useState([])
+
+  useEffect(() => {
+    fetchThoughts()
+  }, [])
```
Comment (Dopest0727) [inline code/src/components/HappyThoughts.js:17]: Nice use of useState

File: code/src/components/HappyThoughts.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+      },
+      body: JSON.stringify({ message: newThought }),
+    }
+
+    fetch(API_THOUGHTS, option)
+      .then((res) => res.json())
+      .then(() => {
+        fetchThoughts()
+        setNewThought("")
+      })
```
Comment (Dopest0727) [inline code/src/components/HappyThoughts.js:56]: Awesome

File: code/src/index.css [hunk 7]
```diff
@@ -11,3 +11,113 @@ code {
...
+  }
+
+  .heartbutton {
+    vertical-align: middle;
+
+  }
+
+
+
+}
```
Comment (Dopest0727) [inline code/src/index.css:123]: Overall good styling

**Review by Dopest0727 — COMMENTED** [review]
I stand corrected good sir, this is your Magnus Opus! 
Great code, easy to read. Well documented. I have nothing negative to add.

File: code/src/App.js [hunk 8]
```diff
@@ -1,9 +1,10 @@
-import React from 'react'
+import HappyThougts from "components/HappyThoughts"
+import React from "react"
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
+    <div className="BoxContainer">
+      <HappyThougts />
```
Comment (SofiePellegrini) [inline code/src/App.js:7]: Great with a clean App.js!

File: code/src/components/GetThoughts.js [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+    <>
+      {thougts.map((data) => (
+        <div className="thoughts" key={data._id}>
+          <p>{data.message}</p>
+          <div className="likes">
+            <button
+              className="heartbutton"
+              onClick={() => onPostLike(data._id)}
+            >
+              <span role="img" aria-label="Heart">
```
Comment (SofiePellegrini) [inline code/src/components/GetThoughts.js:16]: Good with the accessibility `aria-label`.

File: code/src/components/HappyThoughts.js [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+
+  /* POST THOUGHTS */
+
+  const [newThought, setNewThought] = useState("")
+
+  const handleNewThoughtChange = (event) => {
+    setNewThought(event.target.value)
+  }
+
+  const onFormSubmit = (event) => {
```
Comment (SofiePellegrini) [inline code/src/components/HappyThoughts.js:41]: Easy to follow the code 'cause of good selfexplanatory naming of the consts etc :)

File: code/src/index.css [hunk 11]
```diff
@@ -11,3 +11,113 @@ code {
...
+ .heartbutton {
+   border-radius: 50%;
+   border: none;
+   margin-right: 3px;
+   vertical-align: sub;
+ }
+
+ .date {
+   float: right;
+   justify-content: end;
```
Comment (SofiePellegrini) [inline code/src/index.css:88]: Same here as above.

**Review by SofiePellegrini — COMMENTED** [review]
Great job this week Rijad! Your code is clean and easy to follow and you  meet all the blue requirements. Keep up the good work!

**Pull request closed without merging**